### PR TITLE
feat: new experiment for feed layout preview

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does

https://github.com/dailydotdev/apps/assets/554874/92244d65-a27b-4104-9118-062c26478359

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| New | change settings  |  |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

AS-383 #done

### Preview domain
https://as-383-feed-layout-preview.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/cards/PlaceholderCard.tsx [hunk 1]
```diff
@@ -17,8 +17,7 @@ export const PlaceholderCard = forwardRef(function PlaceholderCard(
       aria-busy
       className={classNames(
         className,
-        'flex flex-col rounded-16 bg-theme-post-disabled p-2',
-        'min-h-card',
+        'flex min-h-card flex-col rounded-16 bg-background-subtle p-2',
```
Comment (rebelchris) [inline packages/shared/src/components/cards/PlaceholderCard.tsx:20]: This is more inline with how the cards actually look :)

File: packages/shared/src/components/cards/list/PlaceholderList.tsx [hunk 2]
```diff
@@ -24,10 +24,16 @@ export const PlaceholderList = forwardRef(function PlaceholderCard(
         </CardSpace>
       </CardTextContainer>
       <CardSpace className="my-4">
-        <Text className="h-4 w-3/4" />
-        <Text className="mt-2 h-4 w-1/2" />
+        <CardSpace className="flex">
```
Comment (rebelchris) [inline packages/shared/src/components/cards/list/PlaceholderList.tsx:27]: This is more inline with how the cards actually look:
Screenshot 2024-06-18 at 16 20 01

File: packages/shared/src/components/fields/Checkbox.tsx [hunk 3]
```diff
@@ -18,6 +18,7 @@ export interface CheckboxProps extends InputHTMLAttributes<HTMLInputElement> {
   id?: string;
   children?: ReactNode;
   className?: string;
+  checkmarkClassName?: string;
```
Comment (rebelchris) [inline packages/shared/src/components/fields/Checkbox.tsx:21]: Decide to add secondary as it has too many use-cases already and would be a big change

File: packages/shared/src/components/onboarding/OnboardingHeader.tsx [hunk 4]
```diff
@@ -11,14 +11,16 @@ import { OnboardingStep, wrapperMaxWidth } from './common';
 type OnboardingHeaderProps = {
   showOnboardingPage: boolean;
   setAuth: Dispatch<SetStateAction<AuthProps>>;
-  onClickNext: () => void;
+  onClickCreateFeed: () => void;
+  onClickNext?: () => void;
```
Comment (rebelchris) [inline packages/shared/src/components/onboarding/OnboardingHeader.tsx:15]: Same already had some usecases so didn't want to disrupt too much.

**Review by idoshamun — APPROVED** [review]

File: packages/shared/src/components/onboarding/OnboardingHeader.tsx (hunk 4 above)
Comment (nensidosari) [reply packages/shared/src/components/onboarding/OnboardingHeader.tsx:15]: wouldn't they do the same thing?

**Review by nensidosari — APPROVED** [review]
Looks great!

**Pull request merged**